envoy/formatter/http_formatter_context.h

```diff
@@ -127,6 +127,8 @@ class HttpFormatterContext {
    */
   const Tracing::Span& activeSpan() const;
 
+  static constexpr absl::string_view category() { return "http"; }
```

Comment:
AFIK these are usually fully qualified (e.g. envoy.tracers.opentelemetry.samplers)

Reply:
yeah a comment would be great